File: client/trunk/src_lua/app/conf/guild_donationConfig.lua
-- 自动生成，请勿修改 
-- 时间：2016/05/18
-- 21102585@qq.com

local guild_donationConfig = class("guild_donationConfig")
function guild_donationConfig:ctor()
	self.fields = {"key", "content", "consume_type", "consume_value", "vip_limit", "contribution", "guild_exp", "guild_capital"}
	self.datas = {
		[1] = {1, "初级", 1, 100000, 0, 100, 100, 100},
		[2] = {2, "中级", 2, 10, 1, 200, 200, 200},
		[3] = {3, "高级", 2, 50, 2, 1000, 1000, 1000},

	}
end
return guild_donationConfig

File: server/trunk/server/src/config/guild_donation_config.erl
%%%-------------------------------------------------------------------
%%% @author zhengsiying
%%% @doc
%%%		自动生成文件，不要手动修改
%%% @end
%%% Created : 2016/10/12
%%%-------------------------------------------------------------------

-module(guild_donation_config).

-include("common.hrl").
-include("config.hrl").

-compile([export_all]).

get_list_conf() ->
	[ guild_donation_config:get(X) || X <- get_list() ].

get_list() ->
	[1, 2, 3].

get(1) ->
	#guild_donation_conf{
		key = 1,
		consume_type = 1,
		consume_value = 100000,
		vip_limit = 0,
		contribution = 100,
		guild_exp = 100,
		guild_capital = 100,
		counter_id = 10002
	};

get(2) ->
	#guild_donation_conf{
		key = 2,
		consume_type = 2,
		consume_value = 10,
		vip_limit = 1,
		contribution = 200,
		guild_exp = 200,
		guild_capital = 200,
		counter_id = 10003
	};

get(3) ->
	#guild_donation_conf{
		key = 3,
		consume_type = 2,
		consume_value = 50,
		vip_limit = 2,
		contribution = 1000,
		guild_exp = 1000,
		guild_capital = 1000,
		counter_id = 10004
	};

get(_Key) ->
	?ERR("undefined key from guild_donation_config ~p", [_Key]).